src/lib.rs

```diff
@@ -148,6 +148,12 @@ const OID_NAME_CONSTRAINTS :&[u64] = &[2, 5, 29, 30];
 // https://www.iana.org/assignments/smi-numbers/smi-numbers.xhtml#smi-numbers-1.3.6.1.5.5.7.1
 const OID_PE_ACME :&[u64] = &[1, 3, 6, 1, 5, 5, 7, 1, 31];
 
+#[cfg(feature = "pem")]
+const LINE_ENDING: pem::LineEnding = match cfg!(target_family = "windows") {
```

Comment:
Could you make this a `pem::EncodeConfig`, or alternatively, make another const that has declares that `EncodeConfig`? Then one would save on repetition.